### ISSUE TITLE
Valorisation BON ACHAT pour BF/Mises en "Produit Fini"

### ISSUE THREAD

**Issue by lgazeau** [lgi-wines/Extranet-LGI#146]
En BF/Mises (Ex : Mise 194309 du Product 30057) peut être exécuté de 2 façons 

* L'embouteilleur ne fait que la Prestation de Mise en bouteilles, LGI achète Vin + Matières Séches
* L'embouteilleur achète tout, et nous vend un "produit fini"

Le BON D'ACHAT n'est pas géré pour ce 2nd cas

Dans ce cas de figure (achat d'un produit fini) Il nous faudrait un BON D’ACHAT valorisé en "Prix de Revient" uniquement, c'est à dire sans notions de PV et Marge, bien évidemment (confidentiel vis à vis du prestataire embouteilleur)

En gros, une extraction du Formulaire du Produit => voir PJ
 
[Formulaire Product 30057.pdf](https://github.com/lgi-wines/Extranet-LGI/files/1606261/Formulaire.Product.30057.pdf)

**Comment by Portekoi** [comment]
@lgazeau Comment savoir quelle version afficher lors du click sur la génération du BF?

**Comment by lgazeau** [comment]
Dans le **Formulaire du produit** 👍 
* si case "Prestation de service ?" n'est pas cochée => LGI achète à l'embouteilleur une Produit FINI (LGI sera facturée de tous les composants par l'embouteilleur)
* si case "Prestation de service ?" est cochée => L'embouteilleur facturera uniquement la MISE EN BOUTEILLES, LGI achètera les autres composants aux fournisseurs indiqués (LGI sera facturé des autres composants par leurs fournisseurs respectifs).
-
2nde méthode : 
Sur l'onglet "Composant" du Produit, créer tout simplement un BON D'ACHAT "Tous composants" (sans PV et sans MARGES) => cf. PJ
[Formulaire.Product.30057 (1).pdf](https://github.com/lgi-wines/Extranet-LGI/files/1720579/Formulaire.Product.30057.1.pdf)

**Comment by Portekoi** [comment]
Je pige pas : cette page doit bien être imprimée pour être transmise par mail ou est-ce un nouveau formulaire? Appelle moi sinon : 06 15 94 89 12

**Comment by lgazeau** [comment]
Oui, cette page doit bien être imprimée pour être transmise par mail à l'embouteilleur qui établira sa facture sur nos bases de coûts et de calculs.

**Comment by Portekoi** [comment]
Et le lien vers cette page doit se situer sur le formulaire produit, c'est bien ça?

**Comment by lgazeau** [comment]
Oui, onglet "Composant" du Formulaire Produit
voir PJ
[Formulaire.Product.30057 (Onglet Composants).pdf](https://github.com/lgi-wines/Extranet-LGI/files/1723381/Formulaire.Product.30057.Onglet.Composants.pdf)
